cypress/elements/dashboardItem.js

```diff
@@ -0,0 +1,70 @@
+const clickMenuButton = itemUid => {
+    cy.get(`[data-test="dashboard-item-prog-${itemUid}"]`)
+        .find('[data-test="item-menu-button"]')
+        .click()
+}
+
+const clickViewAsChart = () => {
+    cy.contains('View as Chart').click()
+}
+
+const clickViewAsTable = () => {
+    cy.contains('View as Table').click()
+}
+
+const clickViewAsMap = () => {
+    cy.contains('View as Map').click()
+}
+
+const expectChartToExist = itemUid => {
+    if (itemUid) {
+        cy.get(`[data-test="dashboard-item-prog-${itemUid}"]`)
+            .find('.highcharts-container')
+            .should('exist')
+            .and('be.visible')
+    } else {
+        cy.get('.highcharts-container').should('exist').and('be.visible')
+    }
+}
+
+const expectTableToExist = itemUid => {
+    cy.get(`[data-test="dashboard-item-prog-${itemUid}"]`)
+        .find('.pivot-table-container')
+        .should('exist')
+        .and('be.visible')
+}
+
+const expectTableToNotExist = itemUid => {
+    cy.get(`[data-test="dashboard-item-prog-${itemUid}"]`)
+        .find('.pivot-table-container')
+        .should('not.exist')
+}
+
+const expectChartToNotExist = itemUid => {
+    cy.get(`[data-test="dashboard-item-prog-${itemUid}"]`)
+        .find('.highcharts-container')
+        .should('not.exist')
+}
+
+const expectMapToExist = itemUid => {
+    cy.get(`[data-test="dashboard-item-prog-${itemUid}"]`)
+        .find('.dhis2-map-plugin')
+        .should('exist')
+        .and('be.visible')
+}
+
+export const dashboardItem = {
```

Comment:
I like the `action` and `expect` approach, but its a bit confusing since it looks like an object or an instance of a class, but it's not being used as one. Maybe just prefixing the functions with `action` and `expect` will make it easier to understand and use?

Reply:
I mean prefixing and exporting the individual functions, instead of the `export const dashboardItem`

Reply:
Things have changed, so no longer relevant